subgraphs/uniswap-forks/protocols/spookyswap/src/common/handlers/handleReward.ts

```diff
@@ -0,0 +1,102 @@
+import { BigDecimal, BigInt, ethereum, log } from "@graphprotocol/graph-ts";
+import { NetworkConfigs } from "../../../../../configurations/configure";
+import { MasterChefSpookyswap } from "../../../../../generated/MasterChef/MasterChefSpookyswap";
+import { LiquidityPool, _HelperStore } from "../../../../../generated/schema";
+import { BIGINT_FIVE, BIGINT_ONE, BIGINT_ZERO, INT_ZERO, RECENT_BLOCK_THRESHOLD, UsageType, ZERO_ADDRESS } from "../../../../../src/common/constants";
+import { getOrCreateToken } from "../../../../../src/common/getters";
+import { findNativeTokenPerToken, updateNativeTokenPriceInUSD } from "../../../../../src/price/price";
+import { getRewardsPerDay } from "../../../../../src/common/rewards";
+
+export function handleReward(event: ethereum.Event, pid: BigInt, amount: BigInt, usageType: string): void {
+  let masterChefPool = _HelperStore.load(pid.toString());
+  let poolContract = MasterChefSpookyswap.bind(event.address);
+
+  // Create entity to track masterchef pool mappings
+  if (!masterChefPool) {
+    masterChefPool = new _HelperStore(pid.toString());
+    let poolInfo = poolContract.try_poolInfo(pid);
+    let lpTokenAddress = ZERO_ADDRESS;
+    if (!poolInfo.reverted) {
+      lpTokenAddress = poolInfo.value.value1.toHexString();
+    }
+    masterChefPool.valueString = lpTokenAddress;
+    masterChefPool.valueBigInt = BIGINT_ZERO;
+    masterChefPool.save();
+  }
+
+  // Return if pool does not exist - Banana tokens?
+  let pool = LiquidityPool.load(masterChefPool.valueString!);
+  if (!pool) {
+    return;
+  }
+
+  // Update staked amounts
+  if (usageType == UsageType.DEPOSIT) {
+    pool.stakedOutputTokenAmount = pool.stakedOutputTokenAmount!.plus(amount);
+  } else {
+    pool.stakedOutputTokenAmount = pool.stakedOutputTokenAmount!.minus(amount);
+  }
+
+  // Return if you have calculated rewards recently
+  if (event.block.number.minus(masterChefPool.valueBigInt!).lt(BIGINT_FIVE)) {
```

Comment:
Change BIGINT_FIVE to RECENT_BLOCK_THRESHOLD and add RECENT_BLOCK_THRESHOLD to the main constants file. Set it equal to Big Int 5.